docs/quickstart/csharp.md

```diff
@@ -85,16 +89,16 @@ From the `examples/csharp/helloworld-from-cli` directory:
 > dotnet build **/project.json
 ```
 
-### Using Xamarin Studio or Monodevelop IDEs
-The C# gRPC package currently depends on System.Interactive.Async 3.0.0, which cannot be downloaded on older NuGet versions.
-NuGet is too old in Xamarin Studio on OS X and Monodevelop on Linux as well as via apt-get.
+### Using the Monodevelop IDE
+The C# gRPC package currently depends on System.Interactive.Async 3.0.0, which requires NuGet 2.12+ to install.
+The NuGet included on the latest versions of Monodevelop is too old to install gRPC C#.
```

Comment:
yes this is what I meant, tried to re-word to make this more clear in latest commit